gr-fec/grc/ldpc_decoder_def_list.block.yml

```diff
@@ -44,7 +44,7 @@ templates:
         % elif int(ndim)==1:
         self.${id} = ${id} = list(map( (lambda a: fec.ldpc_decoder.make(${file}, ${sigma}, ${max_iter})), range(0,${dim1})))
         % else:
-        self.${id} = ${id} = list(map( (lambda b: map( ( lambda a: fec.ldpc_decoder.make(${file}, ${sigma}, ${max_iter})), range(0,${dim2}) ) ), range(0,${dim1})))
+        self.${id} = ${id} = list(map( (lambda b: list( map( ( lambda a: fec.ldpc_decoder.make(${file}, ${sigma}, ${max_iter})), range(0,${dim2}) ) ) ), range(0,${dim1})))
```

Comment:
this would be a nice place to use a list generator expression, but for now it's an improvement.